helm/ph-ee-engine/values.yaml

```diff
@@ -1431,6 +1431,9 @@ importer_rdbms:
       host: "operationsmysql"
       port: 3306
       schema: "tenants"
+  bucket_name: "paymenthub-ee"
+  aws:
+    region: "<region>"
```

Comment:
Lets keep the region as a hardocded value (ap-south-1) in G2p Sandbox chart. (NOT HERE)Please create a back log issue to move region into the secret file so that it can be managed externally

Reply:
I am already having a open PR which is adding region through helm chart.

https://github.com/openMF/ph-ee-env-labs/pull/197